File: Python/1043 - Triângulo.py
a,b,c = map(float,input().split(" "))

if (a+b>c and a+c>b and b+c>a):
    d = a+b+c
    print(f"Perimetro = {d:.1f}")

else:
    d = ((a+b)*c)/2
    print(f"Area = {d:.1f}")

File: JavaScript/1043 - Triângulo.js
var [A, B, C] = require('fs').readFileSync('/dev/stdin', 'utf8').split(' ').map(Number);

if (A < B + C && B < A + C && C < A + B) {
    console.log(`Perimetro = ${(A + B + C).toFixed(1)}`);
} else {
    console.log(`Area = ${((A + B) * C / 2).toFixed(1)}`);
}